packages/repo-tools/src/commands/generate-catalog-info/generate-catalog-info.ts

```diff
@@ -144,7 +144,7 @@ function fixCatalogInfoYaml(options: FixOptions) {
     throw new Error(`Unable to parse ${relativePath('.', yamlPath)}: ${e}`);
   }
 
-  const badOwner = !possibleOwners.includes(yamlJson.spec?.owner);
+  const badOwner = possibleOwners[0] !== yamlJson.spec?.owner; // we want the first codeowner to take precedence
```

Comment:
Hmm...  Do we always want this to be the case, or does it make sense just for the initial run / seeding of `catalog-info.yaml` files, @javoire?

This change could require a bit of cleanup of `CODEOWNERS` in this repo, for example.

Reply:
We'll, I think it might still be valuable, just only on creation. So like revert the change in this file, but keep the change in the other file.

Reply:
Ah, I see. Yeah that makes sense, I can make a new PR with that